client/my-sites/earn/components/commission-fees.tsx

```diff
@@ -19,7 +19,7 @@ const CommissionFees = ( { className, commission, iconSize = 16 }: CommissionFee
 					'On your current plan, WordPress.com charges {{em}}%(commission)s{{/em}}.{{br/}} Additionally, Stripe charges are typically %(stripe)s. {{a}}Learn more{{/a}}',
 					{
 						args: {
-							commission: '' + parseFloat( commission ) * 100 + '%',
+							commission: '' + parseFloat( String( commission ) ) * 100 + '%',
```

Comment:
It's interesting that we take a number, parse it to a string, and then to a float again 🤔 Pre-existing, so no need to change anything, but it felt odd to me.

Reply:
Yeah, in theory if the types are right this `parseFloat` is not necessary, but the types are probably wrong since this is here 😩. Adding this double-casting should preserve the existing behavior so that if `commission` is a string, it will be properly converted back to a number.